goalgorilla/open_social
# Feature/3265651 emoji

## Problem
Missing possibility to add emoji to posts and comment 

## Solution
Create social_emoji module for emoji functionality. Add emoji-picker library nolanlawson/emoji-picker-element. Alter widget type for posts, comments and ckeditor to add emoji-picker trigger element.

## Issue tracker
https://www.drupal.org/project/social/issues/3265651

## How to test
- Enable social_emoji module
-  As any user
- Click 'emoji' element near the comment/post/ckeditor
- Emoji picker apper
- Click on any emoji
- It appears in input field

## Screenshots
<img width="668" alt="image" src="https://user-images.githubusercontent.com/94042304/157419004-9fe463af-2699-49a8-b913-d8fbd061d797.png">
